strr-web/package.json

```diff
@@ -43,8 +44,8 @@
     "sass": "^1.71.1",
     "sass-loader": "^13.3.3",
     "typescript": "5.3.3",
-    "vite": "5.2.0-beta.1",
-    "vitest": "1.3.1",
+    "vite": "5.4.6",
+    "vitest": "1.6.0",
```

Comment:
@rstens I updated the version up until the breaking changes were introduced. Any further updates would require a migration.